httpapi/userapi/transfer_details_api.go

```diff
@@ -0,0 +1,14 @@
+package userapi
+
+import (
+	logging "github.com/ipfs/go-log"
+)
+
+var transfersLog = logging.Logger("tranfers-api")
```

Comment:
```suggestion
var log = logging.Logger("user-api")
```

Reply:
TransferAPIService in `transferdetails` package will have its own logger and UserAPI logger is used for all USerAPI logs